windows/src/engine/keyman32/K32_load.cpp

```diff
@@ -75,12 +75,78 @@ BOOL GetKeyboardFileName(LPSTR kbname, LPSTR buf, int nbuf)
   return n;
 }
 
+
+BOOL LoadlpKeyboardCore(int i)
+{
+  SendDebugMessageFormat(0, sdmAIDefault, 0, "LoadlpKeyboardCore: Enter ---");
+  PKEYMAN64THREADDATA _td = ThreadGlobals();
+  if (!_td) return FALSE;
+  if (_td->lpKeyboards[i].coreKeyboard) return TRUE;
+  if (_td->lpActiveKeyboard == &_td->lpKeyboards[i]) _td->lpActiveKeyboard = NULL;  // I822 TSF not working
+
+  char buf[256];
+  if (!GetKeyboardFileName(_td->lpKeyboards[i].Name, buf, 255)) return FALSE;
+  size_t charSize = strlen(buf) + 1;
+
+  LPWSTR keyboardPath = new WCHAR[charSize];
+
+  // Convert char* string to a wchar_t* string.
+  size_t convertedChars = 0;
+  if (mbstowcs_s(&convertedChars, keyboardPath, charSize, buf, _TRUNCATE)) {
+    //goto ExitError;
+    delete[] keyboardPath;
+    return FALSE;
+  }
```

Comment:
Can we just use `strtowstr()` here so we have a single conversion function in use?

Reply:
sure I didn't realise this function exists - will use it. I only found the kmx one which used the std lib strings.